packages/flutter/lib/src/material/stepper.dart

```diff
@@ -207,6 +207,8 @@ class Stepper extends StatefulWidget {
     this.controlsBuilder,
     this.elevation,
     this.margin,
+    this.connectorColor,
+    this.connectorThickness = 1.0,
```

Comment:
Prefer to have the parameter's default be null, so that in the future a  StepperTheme could override the default.

Reply:
In this case, the user must provide a value to `connectorThickness`, right?

Reply:
No, the parameter is optional. In the place that uses the value: `widget.connectorThickness ?? 1.0`.

Reply:
Ok, I got it.